Doc/using/windows.rst

```diff
@@ -23,7 +23,7 @@ available for application-local distributions.
 
 As specified in :pep:`11`, a Python release only supports a Windows platform
 while Microsoft considers the platform under extended support. This means that
-Python |version| supports Windows 8.1 and newer. If you require Windows 7
+Python |version| supports Windows 10 and newer. If you require Windows 7
 support, please install Python 3.8.
```

Comment:
Maybe say something for Windows 8 support instead of Windows 7 support? I don't know if the policy is to mention the previous Windows version and the Python version it supports.

Reply:
It seems Windows 8 was last supported in Python 3.11. [Python 3.12 was released on Oct 2, 2023](https://docs.python.org/3.12/whatsnew/3.12.html), while Windows 8.1 ESU was discontinued in January of 2023.

I think it would be easiest to just remove that sentence than constantly keeping it up-to-date.

Reply:
It's no real difficulty to keep it up to date. Maybe change it to "If you require an earlier version of Windows, please install Python `<version before when we're updating the doc>`"